src/pkg/bundle/deploy.go

```diff
@@ -464,3 +479,34 @@ func setTemplatedVariables(templatedVariables string, pkgVars map[string]string)
 	})
 	return replacedValue
 }
+
+// verifyAndAddFileTo
+func (b *Bundle) verifyAndAddFileTo(helmFileVals []string, filePath string, key string) ([]string, error) {
+	verifiedPath, err := formAndCheckFilePath(b.cfg.DeployOpts.Config, filePath)
+	if err != nil {
+		return nil, err
+	}
+	helmVal := fmt.Sprintf("%s=%v", key, verifiedPath)
+	return append(helmFileVals, helmVal), nil
+}
+
+// formAndCheckFilePath merges relative paths together to form full path and checks if the file exists
+func formAndCheckFilePath(anchorPath string, filePath string) (string, error) {
```

Comment:
Are we handling the case of a user setting the `default` value as opposed to using a `uds-config` or `--set`?

Reply:
yep, is handled as part of `processOverrideVariables()`